for-upstream/common.nix

```diff
@@ -44,7 +50,7 @@ in
       type = types.attrsOf types.unspecified;
       default = {};
     };
-    binaryCachesFile = mkOption {
+    binaryCachesPath = mkOption {
```

Comment:
This will be ambiguous when searching the docs.

Reply:
For example, they google it and find the TOML docs, which is not complete, as this option has more behavior than that option.